src/_scss/pages/search/results/visualizations/rank/_topSection.scss

```diff
@@ -1,18 +1,45 @@
 .visualization-top {
-    @include row();
-    margin-bottom: rem(20);
-    @import "components/visualizations/_description";
-    @import "components/visualizations/_period";
-
-    .no-scope {
-        width: 100%;
+    padding-bottom: rem(20);
+    clear: both;
```

Comment:
Are we clearing some float element here?

Reply:
Yes, the `<hr>` element had a float-left property which I've removed.